### ISSUE TITLE
Outlining improvements

### ISSUE THREAD

**Issue by marisademeglio** [IDPF/epubtestweb#24]
ideally sub items should have 2 or more items to be worthy of a sub bullet.

**Comment by marisademeglio** [comment]
This issue would be more applicable to the epub-testsuite issues list; however, while I agree that it represents good practice in writing, these books are just intended to demonstrate EPUB features, so that isn't one of our high-priority concerns.
